### DISCUSSION TITLE
[Feature resquest] Blend mode

### DISCUSSION THREAD

**Discussion by Pascal-Pouvereau** [olive-editor/olive#2032] [Ideas]
Could Olive have a way to manage blendmodes (maybe in the Opacity filter)?

**Most useful**
prev = previous image
src = current block image
- **Normal** = src
- **Multiply** = prev * src]
- **Multiply ×2** = prev * src * 2
- **Screen** = 1 - (1 - prev) * (1 - src)
- **Additive** = prev + src

**More formulas**
https://en.wikipedia.org/wiki/Blend_modes
https://www.deepskycolors.com/archivo/2010/04/21/formulas-for-Photoshop-blending-modes.html

**Comment by sobotka** [comment]
This doesn't work.

The assumption that all values are closed domain tristimulus, such as with the screen mode, does not hold on open domain mixing, where the bounds are infinite.
